docs/protocol.md

```diff
@@ -105,63 +105,78 @@ In the table below:
+| $`\sigma \gets^{\$} \text{Sign}(sk, m)`$                  | Sign a message $m$ with the private key $sk$                         |
+| $`b \in \{0,1\} \gets \text{Vfy}(pk, m, \sigma)`$         | Verify a message $m$ and a signature $\sigma$ with a public key $pk$ |
+| $` g^x \gets \text{DH(g, x)}`$                            | Diffie-Hellman exponentiation of private component $x$               |
+| $`r \gets^{\$} \text{Rand}()`$                            | Generate a random value                                              |
+| $`mp \gets \text{Pad}(m)`$                                | Pad a message $m$ to a constant size[^1]                             |
+| $`\varepsilon`$                                           | The empty string                                                     |
+
+### `AKEM`: Authenticated KEM
+
+For $\text{AKEM} = \text{DHKEM}(\text{Group}, \text{KDF})$ with:
+
+- $\text{Group} =$ [X25519][RFC 9180 §7.1]
+- $\text{KDF} =$ [HKDF-SHA256][RFC 9180 §7.1]
+
+| Syntax                                         | Description                                                                                                                                                                        |
+| ---------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| $`(sk, pk) \gets^{\$} \text{KGen}()`$          | Generate keys; for DH-AKEM, $(sk, pk) = (x, \text{DH}(g, x)) = (x, g^x)$                                                                                                           |
+| $`(c, K) \gets^{\$} \text{AuthEncap}(sk, pk)`$ | Encapsulate a ciphertext $c$ and a shared secret $K$ using a (sender's) private key $sk$ and a (receiver's) public key $pk$; for DH-AKEM, $(c, K) = (pkE, K) = (pk, K) = (g^x, K)$ |
+| $`K \gets \text{AuthDecap}(sk, pk, c)`$        | Decapsulate a shared secret $K$ using a (receiver's) private key $sk$, a (sender's) public key $pk$, and a ciphertext $c$; for DH-AKEM, $c = pkE$                                  |
+
+### `pskAPKE`: Pre-shared-key authenticated PKE
+
+For $\text{pskAPKE}[\text{AKEM}, \text{KS}, \text{AEAD}]$ with:
+
+- $\text{AKEM}$ as above
+- $\text{KS} =$ HPKE's [`KeySchedule()`][RFC 9180 §5.1] with [HKDF-SHA256][RFC 9180 §7.2]
+- $\text{AEAD} =$ AES-GCM
+
+| Syntax                                                             | Description                                                |
+| ------------------------------------------------------------------ | ---------------------------------------------------------- |
+| $`(c_1, c_2) \gets^{\$} \text{pskAEnc}(sk, pk, psk, m, ad, info)`$ | Encrypt a message $m$ with associated data $ad$ and $info$ |
+| $`m \gets \text{pskADec}(pk, sk, psk, (c_1, c_2), ad, info)`$      | Decrypt a message $m$ with associated data $ad$ and $info$ |
+
+### `SD-PKE`: SecureDrop PKE
```

Comment:
I think that if we name it SD-PKE we should clarify that it is constructed using HPKE AuthPSK and/or alluded to in the last para of "Hybrid Post-Quantum APKE" in Alwen et al, so that it doesn't immediately read as a brand new construction (ie so it's clear it's using existing libraries and not to be implemented from scratch)?

Reply:
(also same nit as above about the sender or receiver subscript so that it's easier to distinguish the keys from a quick read if that's ok)